main.py

```diff
@@ -795,26 +854,83 @@ async def check_replay(message):
-    }
-    r = requests.post("https://api.wc3stats.com/upload", files=file_dic)
-    print(r.content)
-    return r.status_code
+    } 
+    return requests.post("https://api.wc3stats.com/upload", files=file_dic)
+
+@_client.command()
+async def unsub(ctx,arg1 = None):
+    if (arg1 == "EU" or arg1 == "eu"):
+        await ctx.message.author.remove_roles(_EU_role)
+        await ctx.message.channel.send("EU has been succesfully removed from your roles")
+    if (arg1 == "NA" or arg1 == "na"):
+        await ctx.message.author.remove_roles(_NA_role)
+        await ctx.message.channel.send("NA has been succesfully removed from your roles")
+    if (arg1 == "KR" or arg1 == "kr"):
+        await ctx.message.author.remove_roles(_KR_role)
+        await ctx.message.channel.send("KR has been succesfully removed from your roles")
+
+@_client.command()
+async def sub(ctx,arg1 = None):
+    if (arg1 == "EU" or arg1 == "eu"):
+        await ctx.message.author.add_roles(_EU_role)
+        await ctx.message.channel.send("EU has been succesfully added in your roles")
+    if (arg1 == "NA" or arg1 == "na"):
+        await ctx.message.author.add_roles(_NA_role)
+        await ctx.message.channel.send("NA has been succesfully added in your roles")
+    if (arg1 == "KR" or arg1 == "kr"):
+        await ctx.message.author.add_roles(_KR_role)
+        await ctx.message.channel.send("KR has been succesfully added in your roles")
+
+# @_client.command()
+# async def register(ctx,arg1):
+#     if ctx.message.author.roles[-1] < _guild.get_role(params.GRUNT_ID):
+#         await ensure_display(ctx.channel.send, NO_POWER_MSG)
+#         return
+#     
+#     conn = sqlite3.connect(DB_FILE_PATH)
+#     cursor = conn.cursor()
+# 
+#     #check if name is already registered
+#     sqlquery = "SELECT * FROM Players WHERE ent_name = " + name
+#     cursor.execute(sqlquery)
+#     row = cursor.fetchone()
+#     if row is not None:
+#         conn.close()
+#         await ensure_display(ctx.channel.send, "That ENT name has already been registered, no modification were made")
+#         return
+#     
+#     #check if that player has already registered an ENT name, if so => modify the entry
+#     sqlquery = "SELECT RowID FROM Players WHERE Player_id = " + ctx.message.author.id
+#     cursor.execute(sqlquery)
+#     row = cursor.fetchone()
+#     if row is not None:
+#         #delete the value first
+#         RowID = row[0]
+#         sqlquery = "DELETE FROM Players WHERE RowID = " + str(RowID)
+#         cursor.execute(sqlquery)
+#         conn.commit()
+#     sqlquery = "INSERT INTO Players (Player_id,ent_name) VALUES (" + str(ctx.message.author.id) + "," + name + ")"
+#     cursor.execute(sqlquery)
+#     conn.commit()
+#     conn.close()
 
 
 # ==== LOBBIES =====================================================================================
 
 LOBBY_REFRESH_RATE = 5
 QUERY_RETRIES_BEFORE_WARNING = 10
 ENSURE_DISPLAY_WINDOW = LOBBY_REFRESH_RATE * 2
-BELL_EMOJI = "🔔"
```

Comment:
Everything from here + below was my bell changes, and also the `send_message_with_bell_reactions` at the top.

Other than that, looks good!